flocx_market/cmd/api.py

```diff
@@ -27,3 +27,7 @@ def main():
     server = wsgi_service.WSGIService('flocx_market_api')
     launcher.launch_service(server, workers=server.workers)
     launcher.wait()
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
Use `sys.exit(main())` instead.

Reply:
This only exists so that I do not have to reinstall the package every single time I want to start the server. I will move this code to a module that I will just not stage for commit

Reply:
I think this code is fine! Just do something more like: https://github.com/openstack/ironic/blob/master/ironic/cmd/api.py#L48